src/gui/torrentcreatordialog.ui

```diff
@@ -452,6 +452,7 @@
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
+        <enum>Qt::LeftToRight</enum>
```

Comment:
If this fixes your issue then I suppose Qt gets it wrong internally and it is a Qt bug, not a qbt bug.

Also see: [setLayoutDirection](https://doc.qt.io/qt-5/qt.html#LayoutDirection-enum)
>Right-to-left layouts are necessary for certain languages, notably Arabic and Hebrew.